recipes/nextsilicon-cpp-subprocess/all/conanfile.py

```diff
@@ -0,0 +1,27 @@
+from conans import ConanFile, tools
+
+
+class CppSubprocess(ConanFile):
+    name = "nextsilicon-cpp-subprocess"
+    license = "MIT"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/nextsilicon/cpp-subprocess#readme"
+    topics = ("subprocess", "os", "fork")
+    description = ("Subprocessing with modern C++, "
+                   "The only goal was to develop something that is as close as"
+                   "python subprocess module in dealing with processes.")
+    no_copy_source = True
+    settings = "os", "arch", "compiler", "build_type"
+
+    _source_subfolder = 'cpp-subprocess'
+
+    def source(self):
+        tools.get(**self.conan_data['sources'][self.version],
+                  destination=self._source_subfolder, strip_root=True)
+
+    def package(self):
+        self.copy("subprocess.hpp", dst="include/cpp-subprocess", src=self._source_subfolder)
+        self.copy("LICENSE.MIT", dst="licenses", src=self._source_subfolder)
+
+    def package_id(self):
+        self.info.header_only()
```

Comment:
```suggestion
        self.info.header_only()
        
    def package_info(self):
        self.cpp_info.set_property("cmake_file_name", "cpp-subprocess")
        self.cpp_info.set_property("cmake_target_name", "cpp-subprocess::cpp-subprocess")

        # TODO: to remove in conan v2 once cmake_find_package* generators removed
        self.cpp_info.names["cmake_find_package"] = "cpp-subprocess"
        self.cpp_info.names["cmake_find_package_multi"] = "cpp-subprocess"
```

Reply:
No, don't suggest CMake targets if upstream don't provide config files please. It's already a nightmare to fix all CCI recipes with proper targets.

Reply:
Okay.